scene/gui/popup_menu.cpp

```diff
@@ -531,19 +553,20 @@ void PopupMenu::_draw_items() {
 		item_ofs.x += items[i].h_ofs;
 		if (items[i].separator) {
 			int sep_h = separator->get_center_size().height + separator->get_minimum_size().height;
+			int sep_ofs = Math::floor((h - sep_h) / 2.0);
 			if (text != String()) {
 				int text_size = items[i].text_buf->get_size().width;
 				int text_center = display_width / 2;
 				int text_left = text_center - text_size / 2;
 				int text_right = text_center + text_size / 2;
 				if (text_left > item_ofs.x) {
-					labeled_separator_left->draw(ci, Rect2(item_ofs + Point2(0, Math::floor((h - sep_h) / 2.0)), Size2(MAX(0, text_left - item_ofs.x), sep_h)));
+					labeled_separator_left->draw(ci, Rect2(item_ofs + Point2(0, sep_ofs), Size2(MAX(0, text_left - item_ofs.x), sep_h)));
 				}
 				if (text_right < display_width) {
-					labeled_separator_right->draw(ci, Rect2(Point2(text_right, item_ofs.y + Math::floor((h - sep_h) / 2.0)), Size2(MAX(0, display_width - text_right), sep_h)));
+					labeled_separator_right->draw(ci, Rect2(Point2(text_right, item_ofs.y + sep_ofs), Size2(MAX(0, display_width - text_right), sep_h)));
 				}
 			} else {
-				separator->draw(ci, Rect2(item_ofs, Size2(display_width, sep_h)));
+				separator->draw(ci, Rect2(item_ofs + Point2(0, sep_ofs), Size2(display_width, sep_h)));
```

Comment:
This addition should be removed to be in line with the changes from #45704. Adding this back in makes the separator margin uneven on the top and bottom sides.

![uneven separator](https://user-images.githubusercontent.com/41730826/108061815-944db480-70a4-11eb-8fa4-67d83a63b6b3.gif)

Reply:
That is actually a bug with `StyleBoxLine` itself, that has been solved here: #46082

Reply:
Oh, ok then. It is a minor issue anyway. If it crops up again it can be addressed then.